scripts/darwin.sh

```diff
@@ -61,6 +61,7 @@ for disk in $DISKS; do
     echo "mountpoints:"
     echo "$mountpoints" | while read -r mountpoint ; do
       echo "  - path: \"$mountpoint\""
+      echo "    volumeName: \"$mountpoint\""
```

Comment:
Is that _guaranteed_ though, or is it just common convention? Perhaps there's a more reliable way of getting the volumeName?